gateway/sensor.go

```diff
@@ -188,16 +191,15 @@ func (g *Gateway) receivePackets() {
 			case 1:
 				// received a LORA packet
 				var payload []byte
-				for i := 0; i < numPackets; i++ {
-					if packet.size == 0 {
-						continue
-					}
-					// Convert packet to go byte array
-					for i := 0; i < int(packet.size); i++ {
-						payload = append(payload, byte(packet.payload[i]))
-					}
-					g.handlePacket(ctx, payload)
+				if packet.size == 0 {
+					continue
 				}
+				// Convert packet to go byte array
+				//nolint:intrange
+				for i := 0; i < int(packet.size); i++ {
```

Comment:
you can do i := range packet.size in go 1.23 i believe

Reply:
I tried that and it errored

Reply:
when cast to an int as well?

Reply:
tried it again visual studio shows it as an error but it still builds and works so changed it!

Reply:
ah! your vscode has older golang in its environment.